packages/game-app/src/userGameStatus/sagas/userGameStatus.ts

```diff
@@ -52,7 +53,8 @@ export function* startListenToOnlineStatusSaga() {
 }
 
 function* executeStopListenToOnlineStatus(action: ReturnType<typeof actions.stopListenToOnlineStatus>) {
-  const rtdb: firebase.database.Database = yield select(loadBalancerSelectors.getRTDB);
+  const gameId: string = yield select(gameSelectors.getSelectedGameId);
+  const rtdb: firebase.database.Database = firebase.app(gameId).database();
```

Comment:
sagas should not depend on firebase, can we move it directly into the api files?